src/pages/library/common.js

```diff
@@ -317,6 +316,9 @@ const getContextualSuggestion = ([leftContext, match, rightContext]) => {
   ]
 }
 
+const sendSearchEvent = term => Ajax().Metrics.captureEvent(`${Events.catalogFilter}:search`, { term })
+const debounceSearchEvent = _.debounce(5000, sendSearchEvent)
```

Comment:
5 seconds seems long. It's enough time to search something and then navigate to the details page. Perhaps this could be lowered a bit? Perhaps 1 second.